pkg/apis/org/v1/che_types.go

```diff
@@ -103,118 +100,99 @@ type CheClusterSpecServer struct {
 	// Enables the debug mode for Che server. Defaults to `false`.
 	// +optional
 	CheDebug string `json:"cheDebug,omitempty"`
-	// Comma-separated list of ClusterRoles that will be assigned to che ServiceAccount.
-	// Be aware that che-operator has to already have all permissions in these ClusterRoles to be able to grant them.
+	// A comma-separated list of ClusterRoles that will be assigned to Che ServiceAccount.
+	// Be aware that the che-operator has to already have all permissions in these ClusterRoles to grant them.
 	// +optional
 	CheClusterRoles string `json:"cheClusterRoles,omitempty"`
 	// Custom cluster role bound to the user for the Che workspaces.
 	// The default roles are used if this is omitted or left blank.
 	// +optional
 	CheWorkspaceClusterRole string `json:"cheWorkspaceClusterRole,omitempty"`
-	// Defines Kubernetes default namespace in which user's workspaces are created
-	// if user does not override it.
+	// Defines Kubernetes default namespace in which user's workspaces are created if user does not override it.
 	// It's possible to use <username>, <userid> and <workspaceid> placeholders (e.g.: che-workspace-<username>).
-	// In that case, new namespace will be created for each user (or workspace).
-	// Is used by OpenShift infra as well to specify Project
+	// In that case, new namespace will be created for each user (or workspace). Is used by OpenShift infrastructure as well to specify Project.
 	// +optional
 	WorkspaceNamespaceDefault string `json:"workspaceNamespaceDefault,omitempty"`
-	// Defines if a user is able to specify Kubernetes namespace (or OpenShift project) different from the default.
-	// It's NOT RECOMMENDED to configured true without OAuth configured. This property is also used by the OpenShift infra.
+	// Defines that a user is allowed to specify Kubernetes namespace, or OpenShift project, different from the default.
+	// It's NOT RECOMMENDED to be configured true without OpenShift OAuth configured. The OpenShift infrastructure also uses this property.
```

Comment:
> It's NOT RECOMMENDED to be configured true without OpenShift OAuth configured.

Maybe we should change this sentence, because it uses 'configured' two times. Maybe:

It's NOT RECOMMENDED to set `true` value without OpenShift OAuth configured.

Reply:
make sense